setup.py

```diff
@@ -74,6 +74,31 @@ def run(self):
 pythonPackages.append('ganga/GangaRelease')
 pythonPackages.append('ganga/GangaGUI')
 
+
+install_requires=[
+    'ipython>=5.0.0',
+    'jedi==0.17.2',
+    'httplib2>=0.8',
+    'absl-py>=0.1.2',
+    'google-api-python-client',
+    'google-auth-httplib2',
+    'google-auth-oauthlib',
+    'requests>=2.23.0',
+    'Flask>=1.1.2',
+    'PyJWT~=1.7.1',
+    'Flask-SQLAlchemy>=2.4.3',
+    'gunicorn>=20.0.4',
+    'Flask-Login>=0.5.0',
+    'Flask-SocketIO==4.3.1',
+    'SQLAlchemy==1.3.19',
+    "docker",
+    "pymongo",
+    "gdown",
+]
+if sys.platform != 'darwin':
```

Comment:
This doesn't fill me with joy but I guess its the only practical solution

Reply:
Yes, I looked around. To get a behaviour like "Try to install htcondor if available, but don't fail if not" seems only to be possible with the horrible hacks.